packages/fxa-auth-server/scripts/populate-vat-taxes.ts

```diff
@@ -82,6 +80,12 @@ export async function init() {
           continue;
         }
 
+        // Does the subscription already have the right tax rate attached?
+        const existingTaxRate = sub.default_tax_rates?.shift();
```

Comment:
Not really an issue here given the scope of `sub`, but not a fan of using `shift` as it mutates the array.

Reply:
Me neither, but the `?` syntax is fine with that, but it didn't like array access. 🤷🏻